### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+*.sw[nop]
+*~
+.project
+.settings
+TAGS
+bower_components
+coverage
+node_modules
+reports
+.cache
```
Comment (davidlehn) [inline .gitignore:10]: Not using python or bower.

File: RegisterController.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import {AccountService} from './AccountService.js';
+import pDebounce from 'p-debounce';
+
+export class RegisterController {
+  constructor({debounceExists = 500, accountServiceConfig = {}} = {}) {
+    this.state = {
+      registering: false,
+      checkingExistence: false,
+      email: null,
+      // phoneNumber: null
```
Comment (mattcollier) [inline RegisterController.js:15]: What is going on with `phoneNumber`?

**Review by mattcollier — APPROVED** [review]
One question about `phoneNumber`, after approval, is this ready for a 1.0.0 release?

File: RegisterController.js (hunk 2 above)
Comment (dlongley) [reply RegisterController.js:15]: I made an issue (#3) for it. It can be left alone.

**Comment by dlongley** [comment]
After approval we can do a 1.0.0 release.

**Comment by dlongley** [comment]
Closing -- will need to do a PR from `dev` branch instead.

**Pull request closed without merging**